pkg/runtime/wfengine/backends/actors/workflow_actor.go

```diff
@@ -417,13 +417,24 @@ func (wf *workflowActor) runWorkflow(ctx context.Context, actorID string, remind
 
 	// The logic/for loop below purges/removes any leftover state from a completed or failed activity
 	transactionalRequests := make(map[string][]actors.TransactionalOperation)
+	isWorkflowExecutionStartedEvent := false
+	scheduledStartTimestamp := time.Time{}
```

Comment:
Nit: `time.Time{}` can be confusing (at least it is to me) as one needs to remember that this is not the current time.


```suggestion
	var (
        isWorkflowExecutionStartedEvent bool
	    scheduledStartTimestamp time.Time
    )
```